### DISCUSSION TITLE
Change the generated link

### DISCUSSION THREAD

**Discussion by Finn-Cedric** [pglombardo/PasswordPusher#1453] [Q&A]
Hi everyone,

is it possible to change the marked part of the generated link?

![image](https://github.com/pglombardo/PasswordPusher/assets/145297139/4d9701e2-cb1a-4534-ac82-ea2f6271e7ee)

Thank you!

**Comment by Finn-Cedric** [comment]
Seems like it takes the DNS.

**Reply by pglombardo** [reply]
Hi @Finn-Cedric,

That value is taken from the incoming request headers.  If you are using a front end proxy and pass the `X-Forwarded-*` headers, it will use the domain from those headers.

If for some other reason you just have to force the base domain use `PWP__OVERRIDE_BASE_URL`.  Read [this section in Configuration.md](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#proxies).

I made some assumptions (e.g. that you are self-hosting the application) but I believe this is what you'll need.  If not, please elaborate and I'll do what I can to help out.

**Reply by pglombardo** [reply]
Hi @Finn-Cedric - did this help out at all?

**Reply by Finn-Cedric** [reply]
Hi, sorry for the long wait. We will try this. Thank you for your quick help!
